collection-scripts/targeted_logs

```diff
@@ -84,7 +85,7 @@ for nsvm in ${target_vms[@]}; do
 done
 
 # Collect CDI/importer pod logs for imported DVs
-for nsdv in ${target_dvs[@]}; do
+for nsdv in ${target_dvs[@]} ${target_pvcs[@]}; do
```

Comment:
@liranr23 could you please add few words in the commit message to explain why we do this?

Reply:
i did have some word about it in the commit.

Reply:
ah right but you have two commits so the text doesn't propagate to the description of the PR, I meant the description of the PR...

Reply:
done